airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -843,3 +848,82 @@ def _parse_s3_config(config_file_name: str, config_format: str | None = "boto",
         config_format=config_format,
         profile=profile,
     )
+
+
+class AwsBaseHookAsync(AwsBaseHook):
```

Comment:
Is it necessarily base this class on AwsBaseHook? Seems like it do not actually use any methods and properties of it.